beetmoverscript/src/beetmoverscript/data/artifactMap_beetmover_task_schema.json

```diff
@@ -57,12 +57,7 @@
                         }
```

Comment:
This schema was unused since we switched to declarative artifacts so I cleared it and reused it. I don't like at all how schemas look like currently, it's a mess, but this should unblock for now at least.